WordPress/src/main/res/layout/notifications_detail_activity.xml

```diff
@@ -14,33 +34,17 @@
             android:id="@+id/toolbar_main"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
+
             app:theme="@style/WordPress.ActionBar" />
 
     </com.google.android.material.appbar.AppBarLayout>
 
     <org.wordpress.android.widgets.WPViewPager
         android:id="@+id/viewpager"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        app:layout_behavior="@string/appbar_scrolling_view_behavior" />
-
-    <ProgressBar
-        android:id="@+id/progress_loading"
         android:layout_width="wrap_content"
+        android:background="@color/gray_5"
```

Comment:
Added it to give the transition a background colour for distinction between notification